src/components/Common/Module/FieldItem.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div
+    class="d-flex"
+  >
+    <font-awesome-icon
+      :icon="['fas', 'grip-vertical']"
+      class="align-baseline mt-1 mr-3 text-primary handle"
+    />
+    <b
+      class="text-dark cursor-default"
+    >
+      {{ fields }}
```

Comment:
Isn't this more like fieldLabel ?

Reply:
fieldLabel and fieldName are shown if there is a fieldName. So that's why I named it  fields.

Reply:
Still not sure how this are "fields", isn't this just a "custom" label that is made of label + name ?